pkg/ckecli/cmd/reboot_queue_is_enabled.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cybozu-go/well"
+	"github.com/spf13/cobra"
+)
+
+var rebootQueueIsEnabledCmd = &cobra.Command{
+	Use:   "is-enabled",
+	Short: "show reboot queue status",
+	Long:  `Show reboot queue status.`,
+
+	RunE: func(cmd *cobra.Command, args []string) error {
+		well.Go(func(ctx context.Context) error {
+			disabled, err := storage.IsRebootQueueDisabled(ctx)
+			fmt.Println(!disabled)
+			return err
```

Comment:
Handle `err != nil` before printing.
We cannot say "true" or "false" when `err != nil`.